### PR TITLE
versionist.conf.js: Add versionist config file

### PR DESCRIPTION
**Pull request by floion**
Also adds package.json to include the dependencies required in
versionist.conf.js

Change-type: patch
Signed-off-by: Florin Sarbu <florin@resin.io>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 Change log
 -----------
 
+* Add versionist support [Florin]
```
Comment (nazrhom) [inline CHANGELOG.md]: This will be automatically generated by versionist as `versionist.conf.js: Add versionist config file [Florin]`

Comment (floion) [reply CHANGELOG.md]: Yeah, but I think this will happen only after versionist is merged

Comment (floion) [reply CHANGELOG.md]: Right? ^

Comment (nazrhom) [reply CHANGELOG.md]: Yeah, but I would say to wait for that before merging this, I just want to be sure that the various `versionist.conf.js` files don't have to be changed several times

**Comment by floion** [comment]
@resin-ci retest

**Pull request merged**